adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1058,6 +1130,12 @@ func createRoutes(params *routeCreateParams) (routes []*routev3.Route, err error
 		rewritePath := generateRoutePathForReWrite(basePath, resourcePath, pathMatchType)
 		action.Route.RegexRewrite = generateRegexMatchAndSubstitute(rewritePath, resourcePath, pathMatchType)
 
+		if apiType == "GRPC" {
```

Comment:
does grpc api reach these lines? doesn't it return before line 270

Reply:
It does reach it, tested it again now.